tests/test_timeseriesflattener/test_flattened_dataset/test_add_values.py

```diff
@@ -85,13 +82,12 @@ def test_text_predictor():
         output_spec=TextPredictorSpec(
             values_df=str_to_df(predictor_df_str),
             embedding_fn=bow_test_embedding,
-            dim_reduction_fn=pca_test_embedding,
             lookbehind_days=1,
             resolve_multiple_fn="concatenate",
             fallback=np.NaN,
             feature_name="text_value",
         ),
-        expected_values=[[-15.409306032473708, -0.536602322891607]],
+        expected_values=[[0] * 10],
```

Comment:
```suggestion
        expected_values=[[0] * 10],
```

Are these reasonable values?

Reply:
Yup, the count vectorizer is fit the to synthetic data (which goes something "the patient has ..."). Neither "hello" nor "world" are very are very prevalent in them :)